projects/eval-adxl355-pmdz/chibios_cfg/halconf.h

```diff
@@ -0,0 +1,550 @@
+/*
+    ChibiOS - Copyright (C) 2006..2020 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
```

Comment:
is the `chibios_cfg` folder subject for review? or are these files autogenerated?

Reply:
no, the config files shouldn't be relevant for review, however structural suggestions are welcomed